pkg/cluster/nodes/node.go

```diff
@@ -33,6 +33,14 @@ import (
 	"sigs.k8s.io/kind/pkg/exec"
 )
 
+const (
+	// Docker default bridge network is named "bridge" (https://docs.docker.com/network/bridge/#use-the-default-bridge-network)
+	defaultNetwork = "bridge"
```

Comment:
should it be a constant in the `docker` pkg?

Reply:
probably yes, but this is also mostly a stub to make #484 viable if we go that route so I'm ok to deal with it later.

the docker <-> node coupling needs a lot more work soon, I'm sketching that out in a branch ... 😅